boards/ambiq/apollo3p_evb/Kconfig.defconfig

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2024 Ambiq Micro Inc. <www.ambiq.com>
+
+if BOARD_APOLLO3P_EVB
+
+if BT
```

Comment:
Should this section go in the SOC since the chip is BLE?

Reply:
I moved the default configurations of BT to SOC kconfig. Thanks.